p_sets/2/encryption.rb

```diff
@@ -0,0 +1,43 @@
+module Encryption
+
+  ASCII_LETTERS = {
+    capitals: (65..90),
+    lowercase: (97..122)
+  }
+
+  def cyphertext_reporter(plaintext, integer)
+    puts format('%s', encrypt(plaintext, integer))
```

Comment:
No i was just using it because I learnt it last week from Gareth's review. I'm not sure when to use it and when not. I'm going to read up on it.